config/jobs/kubernetes/sig-node/dra-canary.yaml

```diff
@@ -31,12 +35,8 @@ presubmits:
           curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" kind
           kind build node-image --image=dra/node:latest .
           trap 'kind export logs "${ARTIFACTS}/kind"; kind delete cluster' EXIT
-          # Which DRA features exist can change over time.
-          features=( $(grep '"DRA' pkg/features/kube_features.go | sed 's/.*"\(.*\)"/\1/') )
```

Comment:
BTW, this doesn't work as expected, it seems:
```
$ features=( $(grep '"DRA' pkg/features/kube_features.go | sed 's/.*"\(.*\)"/\1/') ) ; echo $features
DRAAdminAccess
```

I'm going to change it a little bit if you don't mind.
```
$ features="$(grep '"DRA' pkg/features/kube_features.go | sed 's/.*"\(.*\)"/\1/')" ; echo $features
DRAAdminAccess DRAResourceClaimDeviceStatus
```

It can potentially resolve canary job failure "/bin/sh: 6: Syntax error: "(" unexpected", but I'm not 100% sure.